libafl/src/fuzzer/mod.rs

```diff
@@ -490,7 +568,22 @@ where
     ) -> Result<(ExecuteInputResult, Option<CorpusId>), Error> {
         self.evaluate_input_with_observers(state, executor, manager, input, send_events)
     }
-
+    fn add_disabled_input(
+        &mut self,
+        state: &mut Self::State,
+        executor: &mut E,
+        input: <Self::State as UsesInput>::Input,
+    ) -> Result<CorpusId, Error> {
+        let observers = executor.observers();
+        let mut testcase = Testcase::with_executions(input.clone(), *state.executions());
+        testcase.set_disabled(true);
+        // Add the disabled input to the main corpus
+        self.feedback_mut()
+            .append_metadata(state, observers, &mut testcase)?;
```

Comment:
I wonder if the feedback needs information about diabled testcases (or probably better yet, an extra `append_metadata_disabled` fn - since these testcases will never be used, so the metadata is somewhat pointless

Reply:
In that case, we can just ignore it since that is the default behavior anyways